x-pack/elastic-agent/docs/elastic-agent-configuration-example.asciidoc

```diff
@@ -0,0 +1,13 @@
+[[elastic-agent-configuration-example]]
+= Configuration example
+
+experimental[]
+
+The following example shows a full list of configuration options:
+
+//REVIEWERS: I'm not sure whether this is a true statement ^^.
+
+[source,yaml]
+----
+include::elastic-agent_configuration_example.yml[]
+----
```

Comment:
Lets not include that file, I do not think it's up to date, I have added a task to review it.. cc @ruflin

Reply:
I'll comment it out for now.